src/components/pages/Item/ItemPage.tsx

```diff
@@ -20,11 +23,12 @@ const ItemPage = (): JSX.Element => {
   const { itemId } = useParams();
 
   const isAppAnalytics = pathname === buildAppsAnalyticsPath(itemId);
+  const isExportAnalytics = pathname === buildExportAnalyticsPath(itemId);
 
   if (!error && !isLoading) {
     return (
       <>
-        {!isAppAnalytics && (
+        {!isAppAnalytics && !isExportAnalytics && (
```

Comment:
Are these not displayed on different pages ? I do not understand why you need to guard against the paths.

Reply:
Actually YES, I'm trying to not display this within export and app item pages, Any recommendations? 
The thing is this is mutual event for loading and error states, So either to control them this way or to display them in their own pages and them every page has it's own loading and error UI
@spaenleh

Reply:
I see the problem, I think it needs a deeper change, probably involving `ChartsHeader` and `ChartsAlerts` to be moved. Let's create an issue for that.